### ISSUE TITLE
Euporie does not work in Alacritty (without Tmux?)

### ISSUE THREAD

**Issue by Peder2911** [joouha/euporie#18]
Hello again!

I am having a really weird issue, where Euporie doesn't output anything if launched from [Alacritty](https://github.com/alacritty/alacritty), but works fine in gnome-terminal. Also, works fine if run from Tmux within alacritty?

When i run Euporie from "vanilla" Alacritty, there's no output. If i press Ctrl-C, the output is:
```
Traceback (most recent call last):
  File "/home/peder/.local/bin/euporie", line 8, in <module>
    sys.exit(App.launch())
  File "/home/peder/.local/pipx/venvs/euporie/lib/python3.8/site-packages/euporie/app/__init__.py", line 137, in launch
    app = cls()
  File "/home/peder/.local/pipx/venvs/euporie/lib/python3.8/site-packages/euporie/app/__init__.py", line 71, in __init__
    self.configure()
  File "/home/peder/.local/pipx/venvs/euporie/lib/python3.8/site-packages/euporie/app/__init__.py", line 113, in configure
    super().__init__(
  File "/home/peder/.local/pipx/venvs/euporie/lib/python3.8/site-packages/prompt_toolkit/application/application.py", line 314, in __init__
    self._merged_style = self._create_merged_style(include_default_pygments_style)
  File "/home/peder/.local/pipx/venvs/euporie/lib/python3.8/site-packages/euporie/app/__init__.py", line 186, in _create_merged_style
    if self.bg_color:
  File "/home/peder/.local/pipx/venvs/euporie/lib/python3.8/site-packages/euporie/app/term.py", line 181, in bg_color
    if result := _query_term(
  File "/home/peder/.local/pipx/venvs/euporie/lib/python3.8/site-packages/euporie/app/term.py", line 132, in _query_term
    output += stdin.read(1)
KeyboardInterrupt
```

Tmux is an OK workaround for me, just thought I'd let you know.

euporie==0.1.16
alacritty==0.4.1

**Comment by joouha** [comment]
Hello,

I think you're seeing this because alacritty did not implement support for [secondary device attributes](https://github.com/alacritty/alacritty/pull/4011) until version `0.6.0`, which euporie uses to check which features the terminal supports (e.g. sixel images, etc.).

If you update to alacritty `>=0.6.0`, euporie should work. 

Euporie works for me on with `alacritty==0.8.0` on Linux.

Interestingly, I get the exact same issue with `alacritty==0.9.0` on Windows, where it does not seem to respond to secondary device attribute requests. I'm not sure if this is intentional or a bug in alacritty.

---

Euporie should be able to work on terminals without secondary device attribute support: I need to add a timeout such that loading continues if no response from the terminal is received when requesting secondary device attributes.

**Comment by joouha** [comment]
I've added a timeout to the terminal feature detection, so euporie should now work in old versions of alacritty

This should be fixed in the v1.0.0

**Comment by Peder2911** [comment]
Great work! Thanks.
